Libraries/LibGfx/VulkanContext.cpp

```diff
@@ -47,16 +48,52 @@ static ErrorOr<VkPhysicalDevice> pick_physical_device(VkInstance instance)
     devices.resize(device_count);
     vkEnumeratePhysicalDevices(instance, &device_count, devices.data());
 
+#define __EXTENSION_SCORE(name, score) __EXTENSION_SCORE_HELPER(name, score)
+#define __EXTENSION_SCORE_HELPER(name, score) \
+    {                                         \
+        name##sv, score                       \
+    }
+
+    static HashMap<StringView, int> extension_scores {
+#ifdef USE_VULKAN_IMAGES
+        __EXTENSION_SCORE(VK_KHR_EXTERNAL_MEMORY_FD_EXTENSION_NAME, 10),
+        __EXTENSION_SCORE(VK_EXT_IMAGE_DRM_FORMAT_MODIFIER_EXTENSION_NAME, 10),
+#endif
+    };
+
+#undef __EXTENSION_SCORE
+#undef __EXTENSION_SCORE_HELPER
```

Comment:
the following should work too:

```suggestion
    constexpr HashMap<StringView, int>::Entry extension_score(const char* name, int value) {
        return { name""sv, value };
    }

    static HashMap<StringView, int> extension_scores {
#ifdef USE_VULKAN_IMAGES
        extension_score(VK_KHR_EXTERNAL_MEMORY_FD_EXTENSION_NAME, 10),
        extension_score(VK_EXT_IMAGE_DRM_FORMAT_MODIFIER_EXTENSION_NAME, 10),
#endif
    };
```